catroid/src/main/java/org/catrobat/catroid/physics/PhysicsLook.java

```diff
@@ -44,6 +46,12 @@ public PhysicsLook(Sprite sprite, PhysicsWorld physicsWorld) {
 		physicsObject = physicsWorld.getPhysicsObject(sprite);
 	}
 
+	@VisibleForTesting
+	public PhysicsLook(Sprite sprite, PhysicsObject physicsObject) {
```

Comment:
Please try to use `Mockito.spy()` instead of creating a custom constructor. If that doesn't work, let me know